pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1674,7 +1674,7 @@ func (og *operationGenerator) GenerateExpandInUseVolumeFunc(
 
 	eventRecorderFunc := func(err *error) {
 		if *err != nil {
-			og.recorder.Eventf(volumeToMount.Pod, v1.EventTypeWarning, kevents.VolumeResizeFailed, (*err).Error())
+			og.recorder.Eventf(volumeToMount.Pod, nil, v1.EventTypeWarning, kevents.VolumeResizeFailed, "Expanding InUseVolume", (*err).Error())
```

Comment:
@msau42 Should we differentiate between "Expanding Volume" and "Expanding InUseVolume"? Both use the same kubelet event type so I'm wondering if we should maybe use the same string for both?

Reply:
I think it's ok to distinguish. I think one of the purposes of the new API is to be able to allow event type and action to be different.